openapi3.yaml

```diff
@@ -376,12 +570,15 @@ paths:
   /jobs/{jobId}/stage:
     post:
       operationId: addStage
-      summary: Add a new stage to a job
+      summary: Add a new stage to a job with automatic ordering
```

Comment:
instead of automatic ordering i would say:
"as the last stage"
"at the end of the stage array/queue"
something that implies its in the end and not automatic